ssz/codec.py

```diff
@@ -10,11 +10,13 @@
 )
 
 
-def encode(obj):
+def encode(obj, serializer_type=None):
```

Comment:
This should not default to `None` anymore as it's required now.

To be consistent with the terminology, calling it `sedes` would be better I think.